test/dlc_tests/benchmark/sagemaker/huggingface/inference/test_performance_huggingface_inference.py

```diff
@@ -1,5 +1,177 @@
-import pytest
+# Copyright 2019-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import os
 
+import pytest
+import sagemaker
+from sagemaker.model import Model
+from sagemaker.predictor import Predictor
+from sagemaker.serializers import JSONSerializer
+from sagemaker.deserializers import JSONDeserializer
+import time
+import numpy as np
+import signal
+import logging
 
-def dummy_test():
+class TimeoutException(Exception):
     pass
+
+def timeout_handler(signum, frame):
+    raise TimeoutException
+
+@pytest.mark.model("bert-base-uncased-pt")
+@pytest.mark.processor("cpu")
+@pytest.mark.cpu_test
+def test_sm_trained_model_cpu():
+    instance_type = "ml.m5.xlarge"
+    model = "s3://aws-dlc-sample-models/huggingface_models/pytorch/bert-base-uncased-pytorch.tar.gz"
+    task = "fill-mask"
+    input_data = {"inputs": "Hello I'm a [MASK] model."}
+    framework = "pytorch"
+    device = "cpu"
+    latencies = _test_sm_trained_model(instance_type, model, task, input_data, framework, device)
+    assert np.average(latencies) <= 350
```

Comment:
Please configure the expected latency value into the following file:
https://github.com/aws/deep-learning-containers/blob/master/src/benchmark_metrics.py

and get the variable from the file in a similar way to https://github.com/aws/deep-learning-containers/blob/master/test/dlc_tests/benchmark/sagemaker/tensorflow/training/test_performance_tensorflow_sm_training.py#L10-L16.

We may have different latency values based on Transformers versions, or even TF or PT framework versions, and it is better to be prepared for this.

Are both `bert-base-uncased` as well as `roberta-base` expected to have the same latency?